xchange-btcmarkets/src/main/java/org/knowm/xchange/btcmarkets/BTCMarketsAdapters.java

```diff
@@ -277,7 +277,8 @@ public static Trades adaptMarketTrades(
               marketTrade.getTimestamp(),
               marketTrade.getId().toString(),
               null,
-              null);
+              null,
+                  false);
```

Comment:
formatting seams off in many places

Reply:
the was an error on the auto-formating on my IDE i will fix it